rpgJavaInterpreter-core/src/main/kotlin/com/smeup/rpgparser/interpreter/movel.kt

```diff
@@ -0,0 +1,58 @@
+package com.smeup.rpgparser.interpreter
+
+import com.smeup.rpgparser.parsing.ast.AssignableExpression
+import com.smeup.rpgparser.parsing.ast.DataRefExpr
+import com.smeup.rpgparser.parsing.ast.Expression
+import java.math.BigDecimal
+
+private fun assignStringToString(operationExtender: String?, target: DataRefExpr, factor2: Expression, interpreterCoreHelper: InterpreterCoreHelper): Value {
```

Comment:
My question is: would every contributor to this project, present and future, know what operationExtender and factor2 means? If so, good, otherwise we should add a comment